src/module-1/cutStrings/index.js

```diff
@@ -1 +1,11 @@
-export const cutStrings = (arr = []) => {};
+export const cutStrings = (arr = []) => {
+   const result = arr.map(word => {
+      return word.length
+   })
+   let minElm = Math.min(...result)
+   const results = arr.map(word2 => {
+      word2 = word2.slice(0, minElm)
```

Comment:
function arguments mutation is a bad practice
so the new variable should be created here
`const sliced = word.slice(0, minElm)`